lib/Doctrine/ORM/Mapping/ClassMetadataInfo.php

```diff
@@ -1138,12 +1139,11 @@ public function enableAssociationCache($fieldName, array $cache)
 
     /**
      * @param string $fieldName
-     * @param array  $cache
      *
-     * @return mixed[]
+     * @return (int|string)[]
```

Comment:
Are parenthesis understood by IDEs?

Reply:
Understood by VS Code but not by phpstorm. Fixed